### PR TITLE
Creating aws_ec2_managed_prefix_list_entry resource

### PR DESCRIPTION
**Pull request by missingcharacter**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

It does not relate to an open issue or Pull Request

**Note:** I'm changing the `entry` field in `aws_ec2_managed_prefix_list` to be `Computed: true,`, this way if no `entry` is  specified we can use `aws_ec2_managed_prefix_list_entry` to create them. Also, this change does not break tests for `aws_ec2_managed_prefix_list` as shown below.

Output from acceptance testing:

```
$ AWS_DEFAULT_REGION=us-west-2 AWS_PROFILE=my-testing-profile make testacc TESTARGS='-run=TestAccAwsEc2ManagedPrefixList'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccAwsEc2ManagedPrefixList -timeout 180m
=== RUN   TestAccAwsEc2ManagedPrefixListEntry_ipv4
=== PAUSE TestAccAwsEc2ManagedPrefixListEntry_ipv4
=== RUN   TestAccAwsEc2ManagedPrefixListEntry_ipv6
=== PAUSE TestAccAwsEc2ManagedPrefixListEntry_ipv6
=== RUN   TestAccAwsEc2ManagedPrefixListEntry_expectInvalidTypeError
=== PAUSE TestAccAwsEc2ManagedPrefixListEntry_expectInvalidTypeError
=== RUN   TestAccAwsEc2ManagedPrefixListEntry_expectInvalidCIDR
=== PAUSE TestAccAwsEc2ManagedPrefixListEntry_expectInvalidCIDR
=== RUN   TestAccAwsEc2ManagedPrefixListEntry_description
=== PAUSE TestAccAwsEc2ManagedPrefixListEntry_description
=== RUN   TestAccAwsEc2ManagedPrefixListEntry_disappears
=== PAUSE TestAccAwsEc2ManagedPrefixListEntry_disappears
=== RUN   TestAccAwsEc2ManagedPrefixList_basic
=== PAUSE TestAccAwsEc2ManagedPrefixList_basic
=== RUN   TestAccAwsEc2ManagedPrefixList_disappears
=== PAUSE TestAccAwsEc2ManagedPrefixList_disappears
=== RUN   TestAccAwsEc2ManagedPrefixList_AddressFamily_IPv6
=== PAUSE TestAccAwsEc2ManagedPrefixList_AddressFamily_IPv6
=== RUN   TestAccAwsEc2ManagedPrefixList_Entry_Cidr
=== PAUSE TestAccAwsEc2ManagedPrefixList_Entry_Cidr
=== RUN   TestAccAwsEc2ManagedPrefixList_Entry_Description
=== PAUSE TestAccAwsEc2ManagedPrefixList_Entry_Description
=== RUN   TestAccAwsEc2ManagedPrefixList_Name
=== PAUSE TestAccAwsEc2ManagedPrefixList_Name
=== RUN   TestAccAwsEc2ManagedPrefixList_Tags
=== PAUSE TestAccAwsEc2ManagedPrefixList_Tags
=== CONT  TestAccAwsEc2ManagedPrefixListEntry_ipv4
=== CONT  TestAccAwsEc2ManagedPrefixList_disappears
=== CONT  TestAccAwsEc2ManagedPrefixList_Tags
=== CONT  TestAccAwsEc2ManagedPrefixList_Name
=== CONT  TestAccAwsEc2ManagedPrefixListEntry_description
=== CONT  TestAccAwsEc2ManagedPrefixListEntry_disappears
=== CONT  TestAccAwsEc2ManagedPrefixListEntry_expectInvalidCIDR
=== CONT  TestAccAwsEc2ManagedPrefixListEntry_ipv6
=== CONT  TestAccAwsEc2ManagedPrefixList_Entry_Description
=== CONT  TestAccAwsEc2ManagedPrefixList_Entry_Cidr
=== CONT  TestAccAwsEc2ManagedPrefixList_AddressFamily_IPv6
=== CONT  TestAccAwsEc2ManagedPrefixList_basic
=== CONT  TestAccAwsEc2ManagedPrefixListEntry_expectInvalidTypeError
--- PASS: TestAccAwsEc2ManagedPrefixListEntry_expectInvalidTypeError (5.42s)
--- PASS: TestAccAwsEc2ManagedPrefixListEntry_expectInvalidCIDR (6.12s)
--- PASS: TestAccAwsEc2ManagedPrefixList_disappears (27.20s)
--- PASS: TestAccAwsEc2ManagedPrefixList_AddressFamily_IPv6 (32.60s)
--- PASS: TestAccAwsEc2ManagedPrefixList_basic (32.64s)
--- PASS: TestAccAwsEc2ManagedPrefixListEntry_ipv6 (34.40s)
--- PASS: TestAccAwsEc2ManagedPrefixListEntry_disappears (34.69s)
--- PASS: TestAccAwsEc2ManagedPrefixListEntry_description (34.70s)
--- PASS: TestAccAwsEc2ManagedPrefixList_Name (44.52s)
--- PASS: TestAccAwsEc2ManagedPrefixList_Entry_Description (46.54s)
--- PASS: TestAccAwsEc2ManagedPrefixList_Entry_Cidr (48.49s)
--- PASS: TestAccAwsEc2ManagedPrefixList_Tags (55.49s)
--- PASS: TestAccAwsEc2ManagedPrefixListEntry_ipv4 (61.09s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	61.218s
```

### CODE REVIEW

**Comment by missingcharacter** [comment]
Could someone help me re-run this?
image

File: aws/resource_aws_ec2_managed_prefix_list_entry.go [hunk 1]
```diff
@@ -0,0 +1,424 @@
...
+	}
+
+	getEntriesInput := &ec2.GetManagedPrefixListEntriesInput{
+		PrefixListId: pl.PrefixListId,
+	}
+	var entries []*ec2.PrefixListEntry
+	id := prefixListEntryIDHash(pl_id, entry)
+	log.Printf("[DEBUG] Computed EC2 Managed Prefix List entry ID %s", id)
+
+	err = resource.Retry(5*time.Minute, func() *resource.RetryError {
```
Comment (YakDriver) [inline aws/resource_aws_ec2_managed_prefix_list_entry.go:104]: Add a `const` for `5*time.Minute` with a descriptive name in `aws/internal/service/ec2/waiter/waiter.go`.

File: aws/resource_aws_ec2_managed_prefix_list_entry.go [hunk 2]
```diff
@@ -0,0 +1,424 @@
...
+
+	d.SetId(id)
+	return resourceAwsEc2ManagedPrefixListEntryRead(d, meta)
+}
+
+func resourceAwsEc2ManagedPrefixListEntryRead(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).ec2conn
+	pl_id := d.Get("prefix_list_id").(string)
+	pl, err := finder.ManagedPrefixListByID(conn, pl_id)
+	if _, notFound := err.(managedPrefixListNotFound); notFound {
```
Comment (YakDriver) [inline aws/resource_aws_ec2_managed_prefix_list_entry.go:142]: See #16796

File: aws/resource_aws_ec2_managed_prefix_list_entry.go [hunk 3]
```diff
@@ -0,0 +1,424 @@
...
+		pl.Version,
+	)
+
+	return nil
+}
+
+func resourceAwsEc2ManagedPrefixListEntryUpdate(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).ec2conn
+
+	if d.HasChange("description") {
```
Comment (YakDriver) [inline aws/resource_aws_ec2_managed_prefix_list_entry.go:206]: Would it be easier and cause the same user disruption to make `description` `ForceNew`?

File: aws/resource_aws_ec2_managed_prefix_list_entry_test.go [hunk 4]
```diff
@@ -0,0 +1,486 @@
...
+	}
+
+	for _, tc := range cases {
+		actual := prefixListEntryIDHash("pl-12345", tc.Input)
+		if actual != tc.Output {
+			t.Errorf("input: %s \noutput: %s", tc.Input, actual)
+		}
+	}
+}
+
```
Comment (YakDriver) [inline aws/resource_aws_ec2_managed_prefix_list_entry_test.go:43]: Please add a `_disappears` test. See #13527.

File: aws/resource_aws_ec2_managed_prefix_list_entry_test.go [hunk 5]
```diff
@@ -0,0 +1,486 @@
...
+		var parts []string
+		parts = append(parts, plID)
+		parts = append(parts, cidrBlock)
+
+		return strings.Join(parts, "_"), nil
+	}
+}
+
+func testAccAwsEc2ManagedPrefixListEntryIpv4Config(rInt int) string {
```
Comment (YakDriver) [inline aws/resource_aws_ec2_managed_prefix_list_entry_test.go:365]:
```suggestion
func testAccAwsEc2ManagedPrefixListEntryIpv4Config(rName string) string {
```

Here and for configs below, change `rInt int` to `rName string` and adjust configs accordingly.

File: aws/resource_aws_ec2_managed_prefix_list_entry_test.go [hunk 6]
```diff
@@ -0,0 +1,486 @@
...
+	return fmt.Sprintf(`
+resource "aws_ec2_managed_prefix_list" "web" {
+  name           = "terraform_test_%d"
+  address_family = "IPv4"
+  max_entries    = 5
+
+  tags = {
+    Name = "tf-acc-test"
```
Comment (YakDriver) [inline aws/resource_aws_ec2_managed_prefix_list_entry_test.go:373]:
```suggestion
    Name = %[1]q
```

File: aws/resource_aws_ec2_managed_prefix_list_entry_test.go [hunk 7]
```diff
@@ -0,0 +1,486 @@
...
+  }
+}
+
+resource "aws_ec2_managed_prefix_list_entry" "entry_1" {
+  cidr_block     = "::/0"
+  prefix_list_id = aws_ec2_managed_prefix_list.web.id
+}
+`, rInt)
```
Comment (YakDriver) [inline aws/resource_aws_ec2_managed_prefix_list_entry_test.go:400]: `rName`

File: aws/resource_aws_ec2_managed_prefix_list_entry_test.go [hunk 8]
```diff
@@ -0,0 +1,486 @@
...
+}
+
+func testAccAwsEc2ManagedPrefixListEntryDescriptionConfig(rInt int) string {
```
Comment (YakDriver) [inline aws/resource_aws_ec2_managed_prefix_list_entry_test.go:403]: `rName`

File: aws/resource_aws_ec2_managed_prefix_list_entry_test.go [hunk 9]
```diff
@@ -0,0 +1,486 @@
...
+	return fmt.Sprintf(`
+resource "aws_ec2_managed_prefix_list" "web" {
+  name           = "terraform_test_%d"
```
Comment (YakDriver) [inline aws/resource_aws_ec2_managed_prefix_list_entry_test.go:406]: `rName`

File: aws/resource_aws_ec2_managed_prefix_list_entry_test.go [hunk 10]
```diff
@@ -0,0 +1,486 @@
...
+    Name = "tf-acc-test"
+  }
+}
+
+resource "aws_ec2_managed_prefix_list_entry" "entry_1" {
+  cidr_block     = "10.0.0.0/8"
+  description    = "TF acceptance test ec2 managed prefix list entry"
+  prefix_list_id = aws_ec2_managed_prefix_list.web.id
+}
+`, rInt)
```
Comment (YakDriver) [inline aws/resource_aws_ec2_managed_prefix_list_entry_test.go:420]: `rName`

File: aws/resource_aws_ec2_managed_prefix_list_entry_test.go [hunk 11]
```diff
@@ -0,0 +1,486 @@
...
+}
+
+func testAccAwsEc2ManagedPrefixListEntry_updateDescriptionConfig(rInt int) string {
```
Comment (YakDriver) [inline aws/resource_aws_ec2_managed_prefix_list_entry_test.go:423]: `rName`

File: aws/resource_aws_ec2_managed_prefix_list_entry_test.go [hunk 12]
```diff
@@ -0,0 +1,486 @@
...
+	return fmt.Sprintf(`
+resource "aws_ec2_managed_prefix_list" "web" {
+  name           = "terraform_test_%d"
```
Comment (YakDriver) [inline aws/resource_aws_ec2_managed_prefix_list_entry_test.go:426]: `rName`

File: aws/resource_aws_ec2_managed_prefix_list_entry_test.go [hunk 13]
```diff
@@ -0,0 +1,486 @@
...
+  address_family = "IPv4"
+  max_entries    = 5
+
+  tags = {
+    Name = "tf-acc-test"
```
Comment (YakDriver) [inline aws/resource_aws_ec2_managed_prefix_list_entry_test.go:431]: `rName`

File: aws/resource_aws_ec2_managed_prefix_list_entry_test.go [hunk 14]
```diff
@@ -0,0 +1,486 @@
...
+  }
+}
+
+resource "aws_ec2_managed_prefix_list_entry" "entry_1" {
+  cidr_block     = "10.0.0.0/8"
+  description    = "TF acceptance test ec2 managed prefix list entry updated"
+  prefix_list_id = aws_ec2_managed_prefix_list.web.id
+}
+`, rInt)
```
Comment (YakDriver) [inline aws/resource_aws_ec2_managed_prefix_list_entry_test.go:440]: `rName`

File: aws/resource_aws_ec2_managed_prefix_list_entry_test.go [hunk 15]
```diff
@@ -0,0 +1,486 @@
...
+}
+
+func testAccAwsEc2ManagedPrefixListEntryExpectInvalidType(rInt int) string {
```
Comment (YakDriver) [inline aws/resource_aws_ec2_managed_prefix_list_entry_test.go:443]: `rName`

File: aws/resource_aws_ec2_managed_prefix_list_entry_test.go [hunk 16]
```diff
@@ -0,0 +1,486 @@
...
+	return fmt.Sprintf(`
+resource "aws_ec2_managed_prefix_list" "web" {
+  name           = "terraform_test_%d"
```
Comment (YakDriver) [inline aws/resource_aws_ec2_managed_prefix_list_entry_test.go:446]: `rName`

File: aws/resource_aws_ec2_managed_prefix_list_entry_test.go [hunk 17]
```diff
@@ -0,0 +1,486 @@
...
+  address_family = "IPv4"
+  max_entries    = 5
+}
+
+resource "aws_ec2_managed_prefix_list_entry" "ipv6" {
+  cidr_block     = "::/244"
+  prefix_list_id = aws_ec2_managed_prefix_list.web.id
+}
+`, rInt)
```
Comment (YakDriver) [inline aws/resource_aws_ec2_managed_prefix_list_entry_test.go:455]: `rName`

File: aws/resource_aws_ec2_managed_prefix_list_entry_test.go [hunk 18]
```diff
@@ -0,0 +1,486 @@
...
+}
+
+func testAccAwsEc2ManagedPrefixListEntryInvalidIPv4CIDR(rInt int) string {
```
Comment (YakDriver) [inline aws/resource_aws_ec2_managed_prefix_list_entry_test.go:458]: `rName`

File: aws/resource_aws_ec2_managed_prefix_list_entry_test.go [hunk 19]
```diff
@@ -0,0 +1,486 @@
...
+	return fmt.Sprintf(`
+resource "aws_ec2_managed_prefix_list" "foo" {
+  name           = "testing-failure-%d"
```
Comment (YakDriver) [inline aws/resource_aws_ec2_managed_prefix_list_entry_test.go:461]: `rName`

File: aws/resource_aws_ec2_managed_prefix_list_entry_test.go [hunk 20]
```diff
@@ -0,0 +1,486 @@
...
+  address_family = "IPv4"
+  max_entries    = 5
+}
+
+resource "aws_ec2_managed_prefix_list_entry" "ipv4" {
+  cidr_block     = "1.2.3.4/33"
+  prefix_list_id = aws_ec2_managed_prefix_list.foo.id
+}
+`, rInt)
```
Comment (YakDriver) [inline aws/resource_aws_ec2_managed_prefix_list_entry_test.go:470]: `rName`

File: aws/resource_aws_ec2_managed_prefix_list_entry_test.go [hunk 21]
```diff
@@ -0,0 +1,486 @@
...
+}
+
+func testAccAwsEc2ManagedPrefixListEntryInvalidIPv6CIDR(rInt int) string {
```
Comment (YakDriver) [inline aws/resource_aws_ec2_managed_prefix_list_entry_test.go:473]: `rName`

File: aws/resource_aws_ec2_managed_prefix_list_entry_test.go [hunk 22]
```diff
@@ -0,0 +1,486 @@
...
+	return fmt.Sprintf(`
+resource "aws_ec2_managed_prefix_list" "foo" {
+  name           = "testing-failure-%d"
```
Comment (YakDriver) [inline aws/resource_aws_ec2_managed_prefix_list_entry_test.go:476]: `rName`

File: aws/resource_aws_ec2_managed_prefix_list_entry_test.go [hunk 23]
```diff
@@ -0,0 +1,486 @@
...
+  address_family = "IPv6"
+  max_entries    = 5
+}
+
+resource "aws_ec2_managed_prefix_list_entry" "ipv6" {
+  cidr_block     = "::/244"
+  prefix_list_id = aws_ec2_managed_prefix_list.foo.id
+}
+`, rInt)
```
Comment (YakDriver) [inline aws/resource_aws_ec2_managed_prefix_list_entry_test.go:485]: `rName`

**Review by YakDriver — DISMISSED** [review]
@missingcharacter Thank for this PR! I performed a quick review on this and noted some items to fix. If you are able to fix these, that would be great. If not, please let us know so that we can either find someone in the community or take it on ourselves. Also, make sure to check "allow edit from maintainers" to expedite merging.

Unfortunately, we aren't able to get everything done we'd like so you may not hear back from us immediately. Hopefully, we'll be able to get to this in the near future.

**Comment by missingcharacter** [comment]
@YakDriver I will take care of these

File: aws/resource_aws_ec2_managed_prefix_list_entry.go [hunk 24]
```diff
@@ -0,0 +1,424 @@
...
+)
+
+func resourceAwsEc2ManagedPrefixListEntry() *schema.Resource {
+	//lintignore:R011
+	return &schema.Resource{
+		Create: resourceAwsEc2ManagedPrefixListEntryCreate,
+		Read:   resourceAwsEc2ManagedPrefixListEntryRead,
+		Update: resourceAwsEc2ManagedPrefixListEntryUpdate,
+		Delete: resourceAwsEc2ManagedPrefixListEntryDelete,
+		Importer: &schema.ResourceImporter{
```
[the remark this answers is not in the document]
Comment (missingcharacter) [reply aws/resource_aws_ec2_managed_prefix_list_entry.go:24]: I ended up following the pattern `aws/resource_aws_ec2_client_vpn_route.go` and `aws/resource_aws_ec2_client_vpn_authorization_rule.go` use for this part

**Comment by missingcharacter** [comment]
@YakDriver I made the improvements you suggested, acceptance tests still pass and all checks are green

**Comment by missingcharacter** [comment]
Hi @YakDriver am I missing something?

**Comment by missingcharacter** [comment]
@YakDriver Hi, this is been here for a while now, is there something else I should do?

**Comment by missingcharacter** [comment]
@ewbankkit The only lint errors I found were related to secrets manager

image

image

if fixing them would help get this merged, I'll do it

**Comment by ewbankkit** [comment]
@missingcharacter No worries; I can make changes and push to your fork.

**Review by ewbankkit — APPROVED** [review]
LGTM :rocket:.

##### Commercial

```console
% make testacc TESTARGS='-run=TestAccAwsEc2ManagedPrefixListEntry_\|TestAccAwsEc2ManagedPrefixList_'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccAwsEc2ManagedPrefixListEntry_\|TestAccAwsEc2ManagedPrefixList_ -timeout 180m
=== RUN   TestAccAwsEc2ManagedPrefixListEntry_ipv4
=== PAUSE TestAccAwsEc2ManagedPrefixListEntry_ipv4
=== RUN   TestAccAwsEc2ManagedPrefixListEntry_ipv6
=== PAUSE TestAccAwsEc2ManagedPrefixListEntry_ipv6
=== RUN   TestAccAwsEc2ManagedPrefixListEntry_expectInvalidTypeError
=== PAUSE TestAccAwsEc2ManagedPrefixListEntry_expectInvalidTypeError
=== RUN   TestAccAwsEc2ManagedPrefixListEntry_expectInvalidCIDR
=== PAUSE TestAccAwsEc2ManagedPrefixListEntry_expectInvalidCIDR
=== RUN   TestAccAwsEc2ManagedPrefixListEntry_description
=== PAUSE TestAccAwsEc2ManagedPrefixListEntry_description
=== RUN   TestAccAwsEc2ManagedPrefixListEntry_disappears
=== PAUSE TestAccAwsEc2ManagedPrefixListEntry_disappears
=== RUN   TestAccAwsEc2ManagedPrefixList_basic
=== PAUSE TestAccAwsEc2ManagedPrefixList_basic
=== RUN   TestAccAwsEc2ManagedPrefixList_disappears
=== PAUSE TestAccAwsEc2ManagedPrefixList_disappears
=== RUN   TestAccAwsEc2ManagedPrefixList_AddressFamily_IPv6
=== PAUSE TestAccAwsEc2ManagedPrefixList_AddressFamily_IPv6
=== RUN   TestAccAwsEc2ManagedPrefixList_Entry_Cidr
=== PAUSE TestAccAwsEc2ManagedPrefixList_Entry_Cidr
=== RUN   TestAccAwsEc2ManagedPrefixList_Entry_Description
=== PAUSE TestAccAwsEc2ManagedPrefixList_Entry_Description
=== RUN   TestAccAwsEc2ManagedPrefixList_Name
=== PAUSE TestAccAwsEc2ManagedPrefixList_Name
=== RUN   TestAccAwsEc2ManagedPrefixList_Tags
=== PAUSE TestAccAwsEc2ManagedPrefixList_Tags
=== CONT  TestAccAwsEc2ManagedPrefixListEntry_ipv4
=== CONT  TestAccAwsEc2ManagedPrefixList_disappears
=== CONT  TestAccAwsEc2ManagedPrefixList_AddressFamily_IPv6
=== CONT  TestAccAwsEc2ManagedPrefixListEntry_description
=== CONT  TestAccAwsEc2ManagedPrefixListEntry_disappears
=== CONT  TestAccAwsEc2ManagedPrefixListEntry_expectInvalidTypeError
=== CONT  TestAccAwsEc2ManagedPrefixListEntry_expectInvalidCIDR
=== CONT  TestAccAwsEc2ManagedPrefixList_Name
=== CONT  TestAccAwsEc2ManagedPrefixList_Tags
=== CONT  TestAccAwsEc2ManagedPrefixListEntry_ipv6
=== CONT  TestAccAwsEc2ManagedPrefixList_Entry_Cidr
=== CONT  TestAccAwsEc2ManagedPrefixList_basic
=== CONT  TestAccAwsEc2ManagedPrefixList_Entry_Description
--- PASS: TestAccAwsEc2ManagedPrefixListEntry_expectInvalidTypeError (4.91s)
--- PASS: TestAccAwsEc2ManagedPrefixListEntry_expectInvalidCIDR (6.28s)
--- PASS: TestAccAwsEc2ManagedPrefixList_disappears (28.27s)
--- PASS: TestAccAwsEc2ManagedPrefixListEntry_disappears (40.10s)
--- PASS: TestAccAwsEc2ManagedPrefixListEntry_ipv4 (40.50s)
--- PASS: TestAccAwsEc2ManagedPrefixList_AddressFamily_IPv6 (40.59s)
--- PASS: TestAccAwsEc2ManagedPrefixListEntry_ipv6 (43.28s)
--- PASS: TestAccAwsEc2ManagedPrefixList_Entry_Cidr (52.83s)
--- PASS: TestAccAwsEc2ManagedPrefixList_Name (53.17s)
--- PASS: TestAccAwsEc2ManagedPrefixList_Entry_Description (54.53s)
--- PASS: TestAccAwsEc2ManagedPrefixList_basic (61.02s)
--- PASS: TestAccAwsEc2ManagedPrefixListEntry_description (62.49s)
--- PASS: TestAccAwsEc2ManagedPrefixList_Tags (64.05s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	68.051s
```

##### GovCloud

```console
% make testacc TESTARGS='-run=TestAccAwsEc2ManagedPrefixListEntry_\|TestAccAwsEc2ManagedPrefixList_'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccAwsEc2ManagedPrefixListEntry_\|TestAccAwsEc2ManagedPrefixList_ -timeout 180m
=== RUN   TestAccAwsEc2ManagedPrefixListEntry_ipv4
=== PAUSE TestAccAwsEc2ManagedPrefixListEntry_ipv4
=== RUN   TestAccAwsEc2ManagedPrefixListEntry_ipv6
=== PAUSE TestAccAwsEc2ManagedPrefixListEntry_ipv6
=== RUN   TestAccAwsEc2ManagedPrefixListEntry_expectInvalidTypeError
=== PAUSE TestAccAwsEc2ManagedPrefixListEntry_expectInvalidTypeError
=== RUN   TestAccAwsEc2ManagedPrefixListEntry_expectInvalidCIDR
=== PAUSE TestAccAwsEc2ManagedPrefixListEntry_expectInvalidCIDR
=== RUN   TestAccAwsEc2ManagedPrefixListEntry_description
=== PAUSE TestAccAwsEc2ManagedPrefixListEntry_description
=== RUN   TestAccAwsEc2ManagedPrefixListEntry_disappears
=== PAUSE TestAccAwsEc2ManagedPrefixListEntry_disappears
=== RUN   TestAccAwsEc2ManagedPrefixList_basic
=== PAUSE TestAccAwsEc2ManagedPrefixList_basic
=== RUN   TestAccAwsEc2ManagedPrefixList_disappears
=== PAUSE TestAccAwsEc2ManagedPrefixList_disappears
=== RUN   TestAccAwsEc2ManagedPrefixList_AddressFamily_IPv6
=== PAUSE TestAccAwsEc2ManagedPrefixList_AddressFamily_IPv6
=== RUN   TestAccAwsEc2ManagedPrefixList_Entry_Cidr
=== PAUSE TestAccAwsEc2ManagedPrefixList_Entry_Cidr
=== RUN   TestAccAwsEc2ManagedPrefixList_Entry_Description
=== PAUSE TestAccAwsEc2ManagedPrefixList_Entry_Description
=== RUN   TestAccAwsEc2ManagedPrefixList_Name
=== PAUSE TestAccAwsEc2ManagedPrefixList_Name
=== RUN   TestAccAwsEc2ManagedPrefixList_Tags
=== PAUSE TestAccAwsEc2ManagedPrefixList_Tags
=== CONT  TestAccAwsEc2ManagedPrefixListEntry_ipv4
=== CONT  TestAccAwsEc2ManagedPrefixList_Tags
=== CONT  TestAccAwsEc2ManagedPrefixList_Entry_Description
=== CONT  TestAccAwsEc2ManagedPrefixListEntry_disappears
=== CONT  TestAccAwsEc2ManagedPrefixList_basic
=== CONT  TestAccAwsEc2ManagedPrefixList_Entry_Cidr
=== CONT  TestAccAwsEc2ManagedPrefixList_AddressFamily_IPv6
=== CONT  TestAccAwsEc2ManagedPrefixList_Name
=== CONT  TestAccAwsEc2ManagedPrefixListEntry_description
=== CONT  TestAccAwsEc2ManagedPrefixListEntry_expectInvalidTypeError
=== CONT  TestAccAwsEc2ManagedPrefixListEntry_expectInvalidCIDR
=== CONT  TestAccAwsEc2ManagedPrefixListEntry_ipv6
=== CONT  TestAccAwsEc2ManagedPrefixList_disappears
=== CONT  TestAccAwsEc2ManagedPrefixList_basic
    resource_aws_ec2_managed_prefix_list_test.go:358: skipping acceptance testing: InvalidAction: The action DescribeManagedPrefixLists is not valid for this web service.
        	status code: 400, request id: 13046c9e-bdca-40aa-9f49-fa9b4add5380
--- SKIP: TestAccAwsEc2ManagedPrefixList_basic (1.88s)
=== CONT  TestAccAwsEc2ManagedPrefixList_Name
    resource_aws_ec2_managed_prefix_list_test.go:358: skipping acceptance testing: InvalidAction: The action DescribeManagedPrefixLists is not valid for this web service.
        	status code: 400, request id: 961bc4bd-f38c-404c-b7d4-979cf51c17fd
--- SKIP: TestAccAwsEc2ManagedPrefixList_Name (1.89s)
=== CONT  TestAccAwsEc2ManagedPrefixListEntry_expectInvalidCIDR
    resource_aws_ec2_managed_prefix_list_test.go:358: skipping acceptance testing: InvalidAction: The action DescribeManagedPrefixLists is not valid for this web service.
        	status code: 400, request id: ac12e0d5-2498-4cf6-bc3a-7f70463c5798
--- SKIP: TestAccAwsEc2ManagedPrefixListEntry_expectInvalidCIDR (1.89s)
=== CONT  TestAccAwsEc2ManagedPrefixList_Entry_Cidr
    resource_aws_ec2_managed_prefix_list_test.go:358: skipping acceptance testing: InvalidAction: The action DescribeManagedPrefixLists is not valid for this web service.
        	status code: 400, request id: 4b815dd2-a685-4a01-8207-5dd29fd0d0c0
--- SKIP: TestAccAwsEc2ManagedPrefixList_Entry_Cidr (1.89s)
=== CONT  TestAccAwsEc2ManagedPrefixListEntry_expectInvalidTypeError
    resource_aws_ec2_managed_prefix_list_test.go:358: skipping acceptance testing: InvalidAction: The action DescribeManagedPrefixLists is not valid for this web service.
        	status code: 400, request id: dcd3fd5e-0d61-4e51-8849-0ec720d7bd42
--- SKIP: TestAccAwsEc2ManagedPrefixListEntry_expectInvalidTypeError (1.89s)
=== CONT  TestAccAwsEc2ManagedPrefixList_Entry_Description
    resource_aws_ec2_managed_prefix_list_test.go:358: skipping acceptance testing: InvalidAction: The action DescribeManagedPrefixLists is not valid for this web service.
        	status code: 400, request id: 8fb46be3-4fc1-4028-82a0-e425b2606e6b
--- SKIP: TestAccAwsEc2ManagedPrefixList_Entry_Description (1.90s)
=== CONT  TestAccAwsEc2ManagedPrefixList_Tags
    resource_aws_ec2_managed_prefix_list_test.go:358: skipping acceptance testing: InvalidAction: The action DescribeManagedPrefixLists is not valid for this web service.
        	status code: 400, request id: 2c9fc8ad-1cb8-44d1-b352-ae7fc1af1282
--- SKIP: TestAccAwsEc2ManagedPrefixList_Tags (1.90s)
=== CONT  TestAccAwsEc2ManagedPrefixList_AddressFamily_IPv6
    resource_aws_ec2_managed_prefix_list_test.go:358: skipping acceptance testing: InvalidAction: The action DescribeManagedPrefixLists is not valid for this web service.
        	status code: 400, request id: adf15578-6414-425c-ab7e-a1204e4767c1
--- SKIP: TestAccAwsEc2ManagedPrefixList_AddressFamily_IPv6 (1.90s)
=== CONT  TestAccAwsEc2ManagedPrefixListEntry_ipv4
    resource_aws_ec2_managed_prefix_list_test.go:358: skipping acceptance testing: InvalidAction: The action DescribeManagedPrefixLists is not valid for this web service.
        	status code: 400, request id: 5151a445-f13b-4d35-9109-7abc87b9ade2
--- SKIP: TestAccAwsEc2ManagedPrefixListEntry_ipv4 (1.90s)
=== CONT  TestAccAwsEc2ManagedPrefixListEntry_description
    resource_aws_ec2_managed_prefix_list_test.go:358: skipping acceptance testing: InvalidAction: The action DescribeManagedPrefixLists is not valid for this web service.
        	status code: 400, request id: e4058b22-1c5e-4de4-8ba2-4e04b9b09997
--- SKIP: TestAccAwsEc2ManagedPrefixListEntry_description (1.91s)
=== CONT  TestAccAwsEc2ManagedPrefixListEntry_disappears
    resource_aws_ec2_managed_prefix_list_test.go:358: skipping acceptance testing: InvalidAction: The action DescribeManagedPrefixLists is not valid for this web service.
        	status code: 400, request id: 024427c2-8789-43f9-a6f8-791defd80399
--- SKIP: TestAccAwsEc2ManagedPrefixListEntry_disappears (1.91s)
=== CONT  TestAccAwsEc2ManagedPrefixListEntry_ipv6
    resource_aws_ec2_managed_prefix_list_test.go:358: skipping acceptance testing: InvalidAction: The action DescribeManagedPrefixLists is not valid for this web service.
        	status code: 400, request id: 9861f174-8ef5-4a5c-91b6-9b63c5d45c6e
--- SKIP: TestAccAwsEc2ManagedPrefixListEntry_ipv6 (1.91s)
=== CONT  TestAccAwsEc2ManagedPrefixList_disappears
    resource_aws_ec2_managed_prefix_list_test.go:358: skipping acceptance testing: InvalidAction: The action DescribeManagedPrefixLists is not valid for this web service.
        	status code: 400, request id: 677e867c-ee3f-4dee-86d5-3bcbc8ff17a6
--- SKIP: TestAccAwsEc2ManagedPrefixList_disappears (1.91s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	6.656s
```

**Comment by ewbankkit** [comment]
Relates: #20991.

**Comment by missingcharacter** [comment]
Thanks @ewbankkit 🙇🏼

**Pull request merged**